portfolio/src/main/webapp/style.css

```
@@ -7,3 +7,108 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.center {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
+}
+body {
+    background-image: url('https://img.freepik.com/free-photo/white-concrete-wall_53876-92803.jpg');
+}
+h1 {text-align: center;}
+*{
+padding:0;
+margin:0;
+text-decoration: none;
+list-style: none;
+box-sizing: border-box;
+}
+
+body{
+    font-family: montserrat;
+}
+nav{
+    background: #0082e6;
+    height: 80px;
+    width: 100%;
+}
+label.logo{
+    color: white;
+    font-size: 35px;
+    line-height: 80px;
+    padding: 0 100px;
+    font-weight: bold;
+}
+nav ul{
+    float: right;
+    margin-right: 20px;
+}
+nav ul li{
+    display: inline-block;
+    line-height: 80px;
+    margin: 0 5px;
+}
+nav ul li a{
+    color: white;
+    font-size: 17px;
+    padding: 7px 13px;
+    border-radius: 3px;
+    text-transform: uppercase;
+}
+a.active,a:hover{
```

[4.2.6](https://google.github.io/styleguide/htmlcssguide.html#Selector_and_Declaration_Separation): Separate selectors and declarations by new lines.

```suggestion
a.active,
a:hover {
```